backend/local/readme.md

```diff
@@ -0,0 +1,11 @@
+start a local dynamo db
```

Comment:
Maybe README.MD in uppercase for consistency, but I would rather prefer to have such information in the main readme.

Reply:
I moved the information into the main readme